config.php

```diff
@@ -33,7 +35,147 @@
+to be effective, we are closing this issue. 
+
+If you want it to be reopened again, feel free to supply us with the requested information.
+
+Thanks!
+
+_This is an automated comment, triggered by adding the label `status:need more info (expired)`._
+COMMENT
+			,
+			'close' => true,
+        ],
+
+        /**
+         * PRs: closed
+         */
+        [
+            'label' => 'pr:too many objectives',
+            'comment' => <<<COMMENT
+Thank you for putting effort in the improvement of our framework. 
+We have reviewed your PR.
+
+We came to the conclusion that this PR in it's current form is covering too many different objectives.
+The bigger a PR is, the more difficult it is to properly assess. 
+
+Please divide your PR in separate PR's with clear objectives.
+
+Thanks!
+
+_This is an automated comment, triggered by adding the label `status:need more info (expired)`._
+COMMENT
+            ,
+            'close' => true,
+        ],
+
+        /**
+         * PRs: kept open
+         */
+        [
+            'label' => 'pr:request for unit tests',
+            'comment' => <<<COMMENT
+Thank you for putting effort in the improvement of our framework. We have reviewed your PR.
+
+In order for the framework and your solution to remain stable in the future, we have a 
+unit test requirement in place. Therefore we can only accept your  PR if it is covered by unit tests. 
+
+Could you add these please?
+
+Thanks!
+
+ps. If you have any questions about the creation of unit tests? Don't hesitate to ask support.
+More information about [unit tests](http://www.yiiframework.com/doc-2.0/guide-test-unit.html)
+
+_This is an automated comment, triggered by adding the label `status:need more info (expired)`._
+COMMENT
+            ,
+        ],
+
+        [
+            'label' => 'pr:loss performance',
+            'comment' => <<<COMMENT
+Thank you for putting effort in the improvement of our framework. 
+We have reviewed your PR.
+
+We came to the conclusion that this PR in current form is going too much at cost of the performance
+of the framework. Speed is an important core value for us. 
+
+Please update your PR so that the performance of the framework is not compromised.
+
+Thanks!
+
+_This is an automated comment, triggered by adding the label `status:need more info (expired)`._
+COMMENT
+            ,
+        ],
+
+        [
+            'label' => 'pr:missing usecase',
+            'comment' => <<<COMMENT
+Thank you for putting effort in the improvement of our framework. 
+We have reviewed your PR.
+
+Unfortunately a usecase is missing. This is required to get a better understanding of the PR.
+It will help us to assess the necessity and applicability in the framework.
+
+Could you supply us with a usecase please? 
+
+Thanks!
+
+_This is an automated comment, triggered by adding the label `status:need more info (expired)`._
+COMMENT
+            ,
+        ],
+
+        [
+            'label' => 'pr:request for 1 commit',
```

Comment:
Not necessary. GitHub has squash-merge now and it's easy to squash when resolving conflicts manually.